### ISSUE TITLE
MAME4All Make error: 127 sysroot_wrapper.hardened: Not found

### ISSUE THREAD

**Issue by LoganTCP** [ValveSoftware/steamlink-sdk#12]
I am running the "build_steamlink.sh" file on PopOs! and running it with "sudo ./build_steamlink.sh".

I don't know what is going on but I do know that a file is not found by the shell but if I look at the folder it's in, it's there.

Thanks,
- LoganTCP

![Screenshot from 2019-03-20 20-07-09](https://user-images.githubusercontent.com/34076740/54715821-40e72a80-4b4c-11e9-98bd-295ad5fdcf36.png)

![Screenshot from 2019-03-20 20-17-07](https://user-images.githubusercontent.com/34076740/54716201-282b4480-4b4d-11e9-962d-22280fc63dba.png)

**Comment by udekmp** [comment]
Yeah I have the same issue with Pop!_OS whenever I try to build anything in the examples.
